api/lib/mno_enterprise/concerns/controllers/jpi/v1/organizations_controller.rb

```diff
@@ -168,6 +168,17 @@ def remove_member
     render 'members'
   end
 
+
+    # POST /mnoe/jpi/v1/organizations/:id/freeze
+    def halt
```

Comment:
Comment is wrong (freeze/halt)
Not sure if `/organization/:id/halt` is the best naming here.

Reply:
`freeze` is reserved by Ruby si I had to choose another name. If someone has a better name I'm all for it.

Reply:
it's reserved for active record, but you can still have an endpoint called freeze. For example : https://github.com/maestrano/maestrano-hub/blob/develop/app/controllers/api/mnoe/v2/organizations_controller.rb#L18